packages/client-password/src/client-password.ts

```diff
@@ -17,16 +18,24 @@ export class AccountsClientPassword {
   /**
    * Create a new user.
    */
-  public async createUser(user: CreateUser): Promise<string> {
-    const hashedPassword = this.hashPassword(user.password);
+  public async createUser(user: PasswordCreateUserType): Promise<string> {
+    let hashedPassword = user.password;
+    if (typeof user.password === 'string') {
+      // mostly just a type check
+      hashedPassword = this.hashPassword(user.password);
```

Comment:
does it is necessary? server stil throw a error if user.password is number, booolean

Reply:
The point of the PR is to fix a possibly null error I believe. It's been a while. But if that's the case hashPassword typings should allow non-string inputs then, which seems a little more messy to me.

Reply:
Right, so the point of this fix is to fix a typescript compile issue. Honestly this fix might not be needed. I forget why I was running into it. Maybe it was just a warning